components/homepage/components/Platform/index.js

```diff
@@ -0,0 +1,39 @@
+import {
+  StPlatformContainer,
+  StPlatformTitle,
+  StPlatformContant,
+  StPlatformContantInfo,
+  StPlatformFadeOne,
+  StPlatformFadeTwo,
+  StPlatformContantInfoBtn,
+  StPlatformContantImage,
+} from "./styles";
+const index = () => {
```

Comment:
same as this and every where that you may created a component with index name is wrong. you should change it to main folder name, and should start with a capital character